notifier.py

```diff
@@ -78,7 +82,10 @@ def alert(url):
     print("{} IN STOCK".format(product))
     print(url)
     webbrowser.open(url, new=1)
-    toast.show_toast("{} IN STOCK".format(product), url, duration=5, icon_path="icon.ico")
+    if NOTIFY_MAC:
+        mac_alert("3080 IN STOCK", url)
```

Comment:
This should refer to the product instead of hardcoded to 3080

Reply:
This has been updated.